File: RazorpayPayment/payment.py
import razorpay

KEY = "rzp_test_4uniVJbkTLvIls"
SECRET = "xdFU73b0zQJtuHMwFbwmcUEi"

client = razorpay.Client(auth=(KEY, SECRET))
DATA = {
    "amount": 29900,
    "currency": "INR",
    "receipt": "finansee_test_payment_receipt",
    "notes": {
        "key1": "value3",
        "key2": "value2"
    }
}
response = client.order.create(data=DATA)
print(response)

File: js/payment.js
// create an order
function CreateOrder() {
  const url = "https://api.razorpay.com/v1/orders";
  data = {
    amount: 29900,
    currency: "INR",
    receipt: "finansee_test_payment",
  };

  fetch(url, {
    method: "POST",
    mode: "cors",
    credentials: "include",
    headers: { "Content-Type": "application/json" },
    Authorization:
      "Basic cnpwX3Rlc3RfNHVuaVZKYmtUTHZJbHM6eGRGVTczYjB6UUp0dUhNd0Zid21jVUVp",
    body: JSON.stringify(data),
  })
    .then((response) => response.json())
    .then((data) => console.log(data));
}
